gdctx/store.go

```diff
@@ -15,14 +19,25 @@ func RegisterStorePrefix(prefix string) {
 }
 
 // InitStore is to initialize the store
-func initStore() {
-	Store = store.New(Restart)
+func InitStore(initPrefix bool) {
+	Store = store.New()
```

Comment:
In this particular case, `Store` variable is exported and consumed by other packages.